aerogear/mobile-security-service-operator
# feat(auth): adding oauth proxy deploy (AEROGEAR-8987)

## Motivation
https://issues.jboss.org/browse/AEROGEAR-8987

## What
Updating resources to add the oauth proxy deploy and configure so that it protects the service

## Why
Security is good

## How
See comments in [this JIRA](https://issues.jboss.org/browse/AEROGEAR-8371) about the manual steps

## Verification Steps
 
I've added verification steps below, but please keep in mind that this is a draft pr for feedback and the route needs to be manually added right now. I will update this later to be done automatically.
I've also removed the Readiness and Liveness Probes to allow the pods to come back up but these should be added back later. 

1. Update docker image to `quay.io/aerogear/mobile-security-service-operator:0.1.0-ag-8987`
2. Run `make create-all
4. Verify that the Service is protected by the proxy and that login/logout works correctly

## Checklist:


## Progress

- [x] Deployment Config
- [x] ServiceAccount
- [x] Route
